static/src/javascripts/projects/common/modules/commercial/sticky-ad-banner.js

```diff
@@ -124,11 +124,12 @@ define([
         var transitionTimingFunction = 'cubic-bezier(0, 0, 0, .985)';
         var transitionDuration = '1s';
 
-        els.$header.css({
+        els.$page.css({
+            'will-change': 'transform',
```

Comment:
Yeah, this is a good point. Here I'm always adding it, so it will definitely be there before the event happens (good), but it won't ever be removed (not good).

We only want to add it when the resize event occurs. When that condition holds true we want to add it one animation frame prior to changing the `transform` value, and we want to remove it one animation frame after the transition is complete. Still thinking about how I can do that in a functional way, which is why this PR is still pending.

As for `will-change` on the ad banner element, good point. I can't say `will-change: transform` though, that would be lying to the browser (and for no need as it's already promoted to a layer thanks to `position: fixed`). I could say `will-change: height`, in which case I'm curious what performance optimisations the browser would make?